diff --git a/Devblocks.class.php b/Devblocks.class.php
index 79e3dac..87e9f8d 100644
--- a/Devblocks.class.php
+++ b/Devblocks.class.php
@@ -609,7 +609,7 @@ class DevblocksPlatform extends DevblocksEngine {
 				"FROM %sextension e ".
 				"INNER JOIN %splugin p ON (e.plugin_id=p.id) ".
 				"WHERE p.enabled = 1 ".
-				"ORDER BY e.plugin_id ASC, e.pos ASC",
+				"ORDER BY e.id ASC, e.plugin_id ASC, e.pos ASC",
 					$prefix,
 					$prefix
 				);
